csfieldguide/utils/custom_converter_templates/interactive.html

```diff
@@ -0,0 +1,22 @@
+{%- if type == 'in-page' -%}
+<remove>
+{{ "{% include 'interactives/" }}{{ name }}{{ ".html' %}" }}
+</remove>
```

Comment:
What are these `<remove>` tags for?

Reply:
no idea, most of these templates bits were copied from the original system

Reply:
have `remove`d them (see what I did there?.... 😉 😆 ) and replaced with a `div`